functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -160,25 +161,32 @@ def get_user(conn, user_id, _redis):
 
 
 def get_user_cache(_redis):
-    try:
-        cache = _redis.get(ALL_USERS_KEY)
-        return cache
-    except TypeError as e:
-        logging.critical('Failed to fetch user from cache: ' + e.args[1])
-        return None
+    if CACHE_TOGGLE == "true":
```

Comment:
I would check with how this variable is set in Azure, and with the other functions. My (human) memory is that the variable was to be "On" or "Off" not "true" and "false." So I save my variable as something like `CACHE_TOGGLE = os.environ['ENV_CACHE_TOGGLE'] == "On"`

Reply:
@unicornsleuth   Weird! I wonder why someone changed it without saying anything. I actually saw your code had it set to 'on' a couple weeks ago and mentioned the difference to Johart. I went with what was in Azure because I couldn't figure out how yours worked.

Reply:
Yeah, I don't know when it was changed in Azure - in the lecture where he creates it in class, my memory is that he decided to make it On/Off. But I've got a PR up to change mine now to match yours!